### ISSUE TITLE
Cosmetizable hacker prefix

### ISSUE THREAD

**Issue by FlyModeZ** [Alexdoru/MWE#55]
cosmetizable hacker prefix
such as &c&l⚠ and &d&l⚠ in this case

![image](https://github.com/user-attachments/assets/8eff5a8c-329e-438a-9d52-0f16a099663e)

(not only the color but also the character are also editable)

**Comment by Alexdoru** [comment]
how do you have a pink icon in pit ?

**Comment by FlyModeZ** [comment]
i drew it lol, just an idea

**Comment by fortXIV** [comment]
this is nice, maybe even option to move it to the right as well?

**Comment by William0216** [comment]
I agree that something like this would be nice, I came here to suggest an option to highlight messages by cheaters, but I like FlyModeZ's idea more. The only change to his idea that I'd suggest is that the icon be left of the name rather than at the start of the line so that the formatting is aligned with normal players' messages since in MW there can be 4 tags.
Ex
[SHOUT] [YELLOW] [ARC] [MVP++] Player_1: message
[SHOUT] [YELLOW] [ZOM] [VIP] ◬ Player_2: message

**Comment by fortXIV** [comment]
bump

**Comment by Alexdoru** [comment]
https://github.com/Alexdoru/MWE/commit/daf45d68a9b36f9a07aa18a106c8930a67c40eb0
